ProjectK.Data/ExperienceRepository.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using ProjectK.Business.ExperiencePoints;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectK.Data
+{
+    public class ExperienceRepository : IExperienceRepository
+    {
+        private readonly ProjectKContext _context;
+
+        public ExperienceRepository(ProjectKContext context)
+        {
+            _context = context;
+        }
+
+
+        public Task<Experience> GetCharacterCurrentXp(int charId)
+        {
+            var querry = _context.Experiences
```

Comment:
querry

Reply:
I am sorry that some English words are hard for me